### ISSUE TITLE
Add UNESCO POIs

### ISSUE THREAD

**Issue by luigifab** [osmandapp/OsmAnd#20288]
### Describe the idea (required)

Could you add UNESCO POIs to OsmAnd?
https://whc.unesco.org/en/list/

### Tell us about the expected behaviour (required)

Currently we can individually find them, but not with UNESCO keyword, and we can't display them like we can display "supermarket" or "bakery".

### Tell us about alternatives you've considered (required)

Are they?

**Comment by scaidermern** [comment]
This site offers a KML file. Did you already try to import it into OsmAnd?

**Comment by luigifab** [comment]
no... a new category like supermarket or shelter is easier to configure!
